### ISSUE TITLE
ansible-lint 6.17.0 does not find any issues on full repo

### ISSUE THREAD

**Issue by MallocArray** [ansible/ansible-lint#3516]
##### Summary

After installing 6.17.0, it scans all files in my directory/repo, but reports 0 failures and 0 warnings when there are known issues in the repo

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 6.17.0 using ansible-core:2.14.6 ruamel-yaml:None ruamel-yaml-clib:None
```

RHEL 8 and Ubuntu 22.04

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

Run only `ansible-lint` against a repo with 1300+ files, with and without excluding a `collections` folder. 

```console (paste below)

```

##### Desired Behavior

using only `ansible-lint` with .ansible-lint file based on the sample file, all valid errors are shown for all folders

##### Actual Behavior

Previous to 6.12, 300+ failures would be detected.  6.12.2 had the issue where it was not excluding directories and also caused a fatal error that prevented it from doing a full run.  6.14 is taking the appropriate time to scan all of the files as expected, but returns 0 issues when we have not addressed all that were previously present.

When using `ansible-lint foldername` it scans just that folder and finds valid issues with files.  Unable to determine exactly what causes it to fail to detect when scanning the entire repo, and no errors are shown in standard mode

Please give some details of what is happening.
Include a [minimum complete verifiable example] with:

- minimized playbook to reproduce the error
- the output of running ansible-lint including the command line used
- if you're getting a stack trace, also the output of
  `ansible-playbook --syntax-check playbook`

Subset of verbose output below

```paste below
...
INFO     Executing syntax check on playbook Netapp/ontap_svmdr_failover.yml (0.87s)
INFO     Executing syntax check on playbook Netapp/ontap_vmware_datastore_nfs.yml (0.95s)
INFO     Executing syntax check on playbook Openshift/create_awx_db_backup_cronjob.yml (0.85s)
INFO     Executing syntax check on playbook Cisco/Intersight/server_info.yml (0.85s)
INFO     Executing syntax check on playbook VMware/drs_groups_and_rules.yml (0.84s)
INFO     Executing syntax check on playbook Netapp/ontap_cifs_share_create.yml (0.93s)
INFO     Executing syntax check on playbook Azure/linux_qualys_extension.yml (0.80s)

DEBUG    Determined rule-profile order: {'internal-error': (0, 'min'), 'load-failure': (1, 'min'), 'parser-error': (2, 'min'), 'syntax-check': (3, 'min'), 'command-instead-of-module': (4, 'basic'), 'command-instead-of-shell': (5, 'basic'), 'deprecated-bare-vars': (6, 'basic'), 'deprecated-local-action': (7, 'basic'), 'deprecated-module': (8, 'basic'), 'inline-env-var': (9, 'basic'), 'key-order': (10, 'basic'), 'literal-compare': (11, 'basic'), 'jinja': (12, 'basic'), 'no-free-form': (13, 'basic'), 'no-jinja-when': (14, 'basic'), 'no-tabs': (15, 'basic'), 'partial-become': (16, 'basic'), 'playbook-extension': (17, 'basic'), 'role-name': (18, 'basic'), 'schema': (19, 'basic'), 'name': (20, 'basic'), 'var-naming': (21, 'basic'), 'yaml': (22, 'basic'), 'name': (23, 'moderate'), 'name': (24, 'moderate'), 'name': (25, 'moderate'), 'spell-var-name': (26, 'moderate'), 'avoid-implicit': (27, 'safety'), 'latest': (28, 'safety'), 'package-latest': (29, 'safety'), 'risky-file-permissions': (30, 'safety'), 'risky-octal': (31, 'safety'), 'risky-shell-pipe': (32, 'safety'), 'galaxy': (33, 'shared'), 'ignore-errors': (34, 'shared'), 'layout': (35, 'shared'), 'meta-incorrect': (36, 'shared'), 'meta-no-tags': (37, 'shared'), 'meta-video-links': (38, 'shared'), 'meta-version': (39, 'shared'), 'meta-runtime': (40, 'shared'), 'no-changed-when': (41, 'shared'), 'no-changelog': (42, 'shared'), 'no-handler': (43, 'shared'), 'no-relative-paths': (44, 'shared'), 'max-block-depth': (45, 'shared'), 'max-tasks': (46, 'shared'), 'unsafe-loop': (47, 'shared'), 'avoid-dot-notation': (48, 'production'), 'sanity': (49, 'production'), 'fqcn': (50, 'production'), 'import-task-no-when': (51, 'production'), 'meta-no-dependencies': (52, 'production'), 'single-entry-point': (53, 'production'), 'use-loop': (54, 'production')}
Passed with production profile: 0 failure(s), 0 warning(s) on 1325 files.
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by MallocArray** [comment]
Seems like it may be related to this collection:
https://github.com/NetApp/santricity

When I have this collection in my /collections/ansible_collections folder in the repo, ansible-lint returns 0 failures and warnings.  If I exclude this folder, like I intend to do with all collections except internally developed ones, then it returns the several hundred failure and warnings I expect.

if I try to scan only this folder, it returns 0 failures and shows only 1 file

```
ansible-lint collections/ansible_collections/netapp_eseries -vvvvv

DEBUG    Folder `collections/ansible_collections/netapp_eseries` does not look like a role due to missing any of the common subfolders such: tasks, meta, vars, defaults, handlers.
INFO     Loading ignores from collections/ansible_collections/netapp_eseries/santricity/.gitignore
DEBUG    Passed 'collections/ansible_collections/netapp_eseries/santricity/changelogs/.plugin-cache.yaml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed 'collections/ansible_collections/netapp_eseries/santricity/changelogs/config.yaml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed 'collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/.travis.yml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed 'collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/.travis.yml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed 'collections/ansible_collections/netapp_eseries/santricity/tests/integration/integration_config.yml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/FILES.json ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/meta/main.yml (meta)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/defaults/main.yml (vars)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_drive_firmware/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_nvme_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/interface/ib_iser.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_auditlog.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_storagepool.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/collect_facts/prefer_embedded.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/.travis.yml (yaml)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_ib_iser_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_discover/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/collect_facts/discovery.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_proxy_firmware_upload.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_storagepool.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_volume.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/collect_facts/validate_system_api_url.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/proxy_security.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_client_certificate.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/CHANGELOG.rst ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_auditlog.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_alerts.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/lun_mapping.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/templates/hostnqn.j2 (jinja2)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_iscsi_interface.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/initiator.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/COPYING ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_syslog.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_facts/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_nvme_interface/tasks/roce.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/collect_facts/prefer_proxy.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/MANIFEST.json ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_proxy_drive_firmware_upload/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/hubPreCheck.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_volume.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/interface.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_proxy_systems/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_iscsi_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_lun_mapping.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_host.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_ldap.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/module_utils/santricity.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_iscsi_target.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/eseries-ansible-collections-diagram.png ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_proxy_drive_firmware_upload.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_lun_mapping.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_discover.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_alerts_syslog.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_lun_mapping.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/changelogs/config.yaml (yaml)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_firmware.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_proxy_drive_firmware_upload.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_server_certificate.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_flashcache.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_ib_iser_interface.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/firmware.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_hostgroup.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_syslog.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_alerts_syslog.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_nvme_interface/tasks/ib.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_auth.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_facts.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/snapshot/rollback.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/meta/main.yml (meta)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_discover.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/changelogs/changelog.yaml (changelog)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/volume.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/README.md ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/README.md ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_ib_iser_interface/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/security.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_iscsi_target.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/interface/nvme_ib.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_client_certificate.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/interface/nvme_roce.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/snapshot/group.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/module_utils/netapp.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/setupBuildParameters.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_asup/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_global.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/doc_fragments/netapp.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_ldap.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_proxy_firmware_upload/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/Jenkinsfile.blackduck ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_iscsi_target.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_volume_copy.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_alerts.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_snapshot_volume.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_firmware/tasks/firmware_tests.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/lookup/santricity_host.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/hubScanProject.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/meta/runtime.yml (meta-runtime)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/.gitignore ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_hostgroup.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_proxy_systems.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/setupBlackduckBuildParameters.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/logging.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/build_info.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/setupSynopsysDetect.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_iscsi_target/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_iscsi_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/hubScanDocker.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_alerts/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_facts.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_facts.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_amg_sync.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_mgmt_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_client_certificate/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_firmware/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_mgmt_interface/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_asup.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/changelogs/.plugin-cache.yaml (yaml)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/snapshot.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_snapshot_group.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_alerts.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_mgmt_interface.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_iscsi_interface/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_auth.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/system.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/defaults/main.yml (vars)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/.travis.yml (yaml)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_storagepool/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_lun_mapping/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_syslog.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_storagepool.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/storage_pool_present.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_auditlog/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_nvme_interface/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_hostgroup.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_proxy_systems.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_auditlog.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/defaults/main.yml (vars)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/lookup/santricity_lun_mapping.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/integration_config.yml (yaml)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_firmware/tasks/firmware_legacy_tests.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_mgmt_interface.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_global.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_auth.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_amg_role.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/README.md ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_drive_firmware.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/interface.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_common/tasks/proxy.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/lookup/santricity_host_detail.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_host.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_proxy_firmware_upload.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_volume/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_amg.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_snapshot_images.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/snapshot/view.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/storage_pool_absent.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/templates/initiatorname_iscsi.j2 (jinja2)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_hostgroup/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_firmware.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/meta/main.yml (meta)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_volume.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/tasks/interface/iscsi.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_management/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/doc_fragments/santricity.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_snapshot.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_host/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/CONTRIBUTING.md ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_syslog/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/lookup/santricity_volume.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_host.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/vars/hubScan.groovy ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_ldap/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_alerts_syslog/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_storage_system.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/na_santricity_asup.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_auth/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/roles/nar_santricity_host/README.md ()
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_nvme_interface.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/integration/targets/na_santricity_global/tasks/main.yml (tasks)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/lookup/santricity_storage_pool.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_drive_firmware.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_asup.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_global.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/tests/unit/modules/test_na_santricity_firmware.py (python)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_drive_firmware.py (plugin)
DEBUG    Excluded collections/ansible_collections/netapp_eseries/santricity/plugins/modules/netapp_e_ldap.py (plugin)

Passed with production profile: 0 failure(s), 0 warning(s) on 1 files.
```

**Comment by ssbarnea** [comment]
Look at line "INFO     Loading ignores from collections/ansible_collections/netapp_eseries/santricity/.gitignore" -- check the content of the file and you will likely find the reason why all those files were excluded.

**Comment by MallocArray** [comment]
I don't understand?  ansible-lint looked through subfolders, found other .gitignores nested inside of folders and then ansible-lint ignored any files it found there?

If anything I would expect that to only impact git and not ansible-lint, but looking at the contents, I don't see why it would cause ansible-lint to still look through the files, but report no issues.

```
*.pyc
.idea
*.iml
ansible.cfg
```

**Comment by ssbarnea** [comment]
I will look at it tomorrow, yep, that content should not cause it to ignore.

Since 6.17.0, all git ignores are loaded by default, recursively, regardless if that is a git repository or not.

Do you also have some exclude paths in linter own config?

**Comment by MallocArray** [comment]
In my `.ansible-lint` file in the root of the project, I do exclude several folders and a few specific files.

Example:
```
exclude_paths:
  # Excluding Galaxy collections as we don't maintain them, but want to check any custom collections
  - collections/ansible_collections/ansible/
  - collections/ansible_collections/awx/
  - collections/ansible_collections/azure/
  - collections/ansible_collections/cisco/
  - collections/ansible_collections/cloud/
  - ```

**Comment by ssbarnea** [comment]
If you cannot share the repo with the problem, please create one similar public repo where the problem can be reproduced. Once we reproduce it, we will be able to fix it, especially as this is a high impact issue.

**Comment by MallocArray** [comment]
Please try this repo:
https://github.com/MallocArray/linting

When I run `ansible-lint` from the root of the project as it is in this repo, I get 0 failures and warnings.  If I edit `.ansible-lint` and uncomment the netapp_eseries line from exclude_paths, and run the same command, I get 4 failures.

**Comment by MallocArray** [comment]
Was the repo helpful?

**Comment by ssbarnea** [comment]
I think that the bug is caused by some deep `.gitignore` files that are affecting files above their level.
